src/jmeter/tests/studentProfile.jmx

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.0 r1840935">
```

Comment:
I think you should change this to 4.0? since gradlew only supports 4.0? Will be good to mention that part in the  docs too